src/main/dist/template/data/resourcemanager/MDR_Plugin_Model_Development_Kit_91110_descriptor.xml

```diff
@@ -1,15 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resourceDescriptor critical="false"
                     date="${build.timestamp}"
-                    description="DocGen, MMS syncing, and common profiles"
+                    description="DocGen, MMS syncing, and common profiles. The extended edition adds pdf generation."
```

Comment:
Can be reverted given it's being rolled into the base edition.

Reply:
done

Reply:
MDR_Plugin_Model_Development_Kit_91110_descriptor.xml and plugin.xml is modified back to base and included OpenMBEE as the provider-name.  Removed the comment out in DocGenUtils.java